server/view/render.js

```diff
@@ -6,10 +6,14 @@ import marked from 'marked';
 export default function render(root) {
   return (ctx, next) => {
     const [flags, cohorts] = ctx.intervalCache.get('flags');
+    const latestTweets = ctx.intervalCache.get('tweets');
+    const latestInstagramPosts = ctx.intervalCache.get('instagramPosts');
     const globals = Map({
       featuresCohort: ctx.featuresCohort,
       featureFlags: flags,
-      cohorts: cohorts
+      cohorts: cohorts,
+      latestTweets: latestTweets,
+      latestInstagramPosts: latestInstagramPosts
```

Comment:
Just wondering whether these should be added to the ctx for every page, or only passed in where they're used, i.e to the explore page?